python/ciqueue/_pytest/outcomes.py

```diff
@@ -14,36 +14,39 @@
 
 from __future__ import absolute_import
 import dill
-from _pytest import runner
+try:
+    from _pytest import outcomes
+except ImportError:
+    from _pytest import runner as outcomes
```

Comment:
This is the big change that we're now supporting. In pytest 3.2, they moved the `Failed` and `Skipped` classes from the `runner` module to `outcomes`.